heron/common/src/python/argparser.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2016 Twitter. All rights reserved.
```

Comment:
I am wondering if we should also name the class to be more generic , say HeronArgumentParser instead of making it specific to HeronRC. this also provides the fallback behavior in case the RC file is not found.